test/csuite/timestamp_abort/main.c

```diff
@@ -1486,9 +1486,6 @@ main(int argc, char *argv[])
             if (num_iterations == 0)
                 num_iterations = 1;
             break;
-        case 'L':
```

Comment:
Not in this file - but the `-L` argument is used in `timestamp_abort/smoke.sh` so those lines need to be removed from there too.

Reply:
Thanks for pointing it out. The 2 command lines in `timestamp_abort/smoke.sh` that make use of the "-L" arg are currently commented out. I'll remove them.

Reply:
See [7915cc6](https://github.com/wiredtiger/wiredtiger/pull/11278/commits/7915cc61f3d937a92907c9ab7892b8e52b3fc448)